IBPSA/Fluid/Humidifiers/SprayAirWasher_X.mo

```diff
@@ -12,9 +12,7 @@ model SprayAirWasher_X
       final QMax_flow = Modelica.Constants.inf,
       final QMin_flow = -Modelica.Constants.inf,
       final mWatMax_flow = mWatMax_flow,
-      final mWatMin_flow = 0,
-      final energyDynamics = Modelica.Fluid.Types.Dynamics.SteadyState,
```

Comment:
Is this removal intentional?

Reply:
Yes, this is intentional as `energyDynamics` is assigned in the base class now (https://github.com/ibpsa/modelica-ibpsa/pull/1591/files#diff-70ded3e6ed7a3a11eee46a4517919a9762e70e4855632fde06dd590b2258ddcfR35). Previously, it was missing in the base class, and extending models corrected this by assigning it one level higher.